### ISSUE TITLE
Driver's Drowsiness Detection system

### ISSUE THREAD

**Issue by dilroseR** [prathimacode-hub/ML-ProjectKart#389]
Define You:

I'm a  DevIncept Participant 

I will create this model using the Transfer Learning technique with the help of MobileNet model

My model will alert the user whenever he/she drowses away with an alarm sound and hence decreasing the road-related accidents

Assign it to me, please.

**Comment by prathimacode-hub** [comment]
Contribution to DevIncept program would count from 10th of July, 2021. And this issue is already taken up by LGMSOC participant. If at all, PR doesn't turn up from the participant by 10th of July, I shall assign this to you. But assigning would take place only after the stipulated data according to the norms of DevIncept Program and is purely on First Come First Serve basis. All the best for your open source journey. @dilroseR

**Comment by dilroseR** [comment]
> Contribution to DevIncept program would count from 10th of July, 2021. And this issue is already taken up by LGMSOC participant. If at all, PR doesn't turn up from the participant by 10th of July, I shall assign this to you. But assigning would take place only after the stipulated date according to the norms of DevIncept Program and is purely on First Come First Serve basis. All the best for your open source journey. @dilroseR

Alright, @prathimacode-hub !
please don't forget to msg if the PR doesn't turn up
Thankyou.

**Comment by prathimacode-hub** [comment]
> > Contribution to DevIncept program would count from 10th of July, 2021. And this issue is already taken up by LGMSOC participant. If at all, PR doesn't turn up from the participant by 10th of July, I shall assign this to you. But assigning would take place only after the stipulated date according to the norms of DevIncept Program and is purely on First Come First Serve basis. All the best for your open source journey. @dilroseR
> 
> Alright, @prathimacode-hub !
> please don't forget to msg if the PR doesn't turn up
> Thankyou.

Yeah, sure. I would let you know. Meanwhile you can taken up an another issue to work on or else you can choose it from issues which are up for grabs. @dilroseR

**Comment by dilroseR** [comment]
> > > Contribution to DevIncept program would count from 10th of July, 2021. And this issue is already taken up by LGMSOC participant. If at all, PR doesn't turn up from the participant by 10th of July, I shall assign this to you. But assigning would take place only after the stipulated date according to the norms of DevIncept Program and is purely on First Come First Serve basis. All the best for your open source journey. @dilroseR
> > 
> > 
> > Alright, @prathimacode-hub !
> > please don't forget to msg if the PR doesn't turn up
> > Thankyou.
> 
> Yeah, sure. I would let you know. Meanwhile you can taken up an another issue to work on or else you can choose it from issues which are up for grabs. @dilroseR

Can i work on this.. as I can see a label called Up for grab?

**Comment by prathimacode-hub** [comment]
Yes, you can work on this. but need some changes form you. Add the issue template and do the changes according to the project above. Then provide with the dataset you'll be using. Let me know of the approach you would follow, task you would be taking up. Once you add up these details, I shall assign this issue to you. @dilroseR

**Comment by dilroseR** [comment]
> Yes, you can work on this. but need some changes form you. Add the issue template and do the changes according to the project above. Then provide with the dataset you'll be using. Let me know of the approach you would follow, task you would be taking up. Once you add up these details, I shall assign this issue to you. @dilroseR

Can you elaborate on add the issue template..? I didn't get that

**Comment by prathimacode-hub** [comment]
There is an issue template provided in README file of the project else you can check up with the assigned issues on how template looks like. There are template to follow for issues, PR and README @dilroseR

**Comment by dilroseR** [comment]
> Yes, you can work on this. but need some changes form you. Add the issue template and do the changes according to the project above. Then provide with the dataset you'll be using. Let me know of the approach you would follow, task you would be taking up. Once you add up these details, I shall assign this issue to you. @dilroseR

**TITLE:** Driver's Drowsiness Detection system

**GOAL:** I aim to build a system which would alert the user whenever the driver falls asleep and hence decreasing thr road related accidents which is mainly caused due to drowsiness.

**DATASET:** http://mrl.cs.vsb.cz/eyedataset

**WHAT WILL I DO:** 
1.) Firstly, I will clean the data so that it can be loaded into the model.
2.) I will be using MobileNet model with the help of Transfer Learning since building from scratch would take up many computation time , resource and power.

**WHY DID I CHOOSE THIS MODEL:** MobileNet is a lightweight model having only 4 million parameters whereas Resnet, GoogleNet has more 18  & 7 million parameters resp. Also the computation time & power taken would be less. This technique helps in increasing the accuracy.

**LIBRARIES NEEDED:** opencv, tensorflow, numpy, matplotlib, flask

**Comment by prathimacode-hub** [comment]
Great, issue assigned. You can add multiple models and compare the efficiency. @dilroseR

**Comment by dilroseR** [comment]
> Great, issue assigned. You can add multiple models and compare the efficiency. @dilroseR

Thankyou
